scripts/install_openssl.sh

```diff
@@ -37,6 +37,7 @@ pushd ${SRCDIR}
 
 # Build the library.
 ${ARCH_PROG} ./config --prefix=${INSTALLDIR} \
+                      --libdir=lib \
```

Comment:
Does this work with a relative name or should it rather be `${INSTALLDIR}/lib` ?

Reply:
Yes, `--libdir` from OpenSSL can take relative paths:

https://github.com/openssl/openssl/blob/5056133cc7d6c52033c25b2e1f7762bcafcce760/INSTALL.md?plain=1#L337-L349

Reply:
:+1: